README.rst

```diff
@@ -12,20 +12,17 @@ pyface: traits-capable windowing framework
     :target: https://codecov.io/github/enthought/pyface?branch=master
 
 
-The pyface project contains a toolkit-independent GUI abstraction layer,
+The Pyface project contains a toolkit-independent GUI abstraction layer,
 which is used to support the "visualization" features of the Traits package.
 Thus, you can write code in terms of the Traits API (views, items, editors,
-etc.), and let pyface and your selected toolkit and back-end take care of
+etc.), and let Pyface and your selected toolkit and back-end take care of
 the details of displaying them.
 
 The following GUI backends are supported:
 
-- wxPython
-- PyQt
-- PySide
-
-**Warning:** The default toolkit if none is supplied is ``qt4``.
-   This changed from ``wx`` in Pyface 5.0..
```

Comment:
There is still a default toolkit and it might be worth mentioning in the README anyway, without the "warning" prefix and the "changed from wx" suffix? This is minor and should not have to block the release though.

Reply:
I decided that it had been long enough that the Qt default doesn't need to be called out in the readme any more (it is talked about in the docs).